{{cookiecutter.project_shortname}}/Pipfile.lock

```diff
@@ -0,0 +1,1650 @@
+{
```

Comment:
I think you're correct (so let's keep the `Pipfile.lock`), but for different reasons. Doing `./scripts/bootstrap` in the beginning runs `pipenv install --dev --skip-lock` which is fast since it doesn't resolve and lock dependencies in `Pipfile.lock`. The problem is that because of the lack of resolving dependencies it might produce a version-conflicted environment (which it actually does at the moment).

I would say that the correct practice would be to actually install dependencies from the `Pipfile.lock` in `bootstrap`, which is equivalent to the old way of first installing from `requirements.txt` before starting to developing the application. This is not slow since it installs pinned dependencies, and produces as stable build. The command for this would be `pipenv install --dev --ignore-pipfile` (or `pipenv sync --dev`) 

Upgrading dependencies should be a separate, conscious step that is initiated by the maintainer of the instance as part of its development process (and probably documented in the official docs)